css/css-images/image-set/image-set-rendering-ref.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Image set rendering</title>
+        <link rel="author" title="Noam Rosenthal" href="mailto:noam@webkit.org">
+        <link rel="help" href="https://drafts.csswg.org/css-images-4/#image-set-notation">
+        <meta name="assert" content="image-set selection by simulated resolution">
+        <meta name="device-pixel-ratio" content="2">
```

Comment:
Does that actually work? If so, what does browser support look like? I'm failing to find a reference...

Reply:
I found it in sourcefile.py:
> return self.root.findall(".//{http://www.w3.org/1999/xhtml}meta[@name='device-pixel-ratio']")

No current tests use it yet. I wanted to have a working test in, and then see how to make this meta tag work in different browsers. I think chrome has OK-ish support for this in command line already.

It's unfortunate that it uses the "dpi" name everywhere in the code when in fact this is about pixel-density (dppx). 

I tested this test on safari nightly with "hardcoding" dpr=2 rather than using the meta tag.

Reply:
@yoavweiss: changed the test to be with DPR=1 for now, it still tests most of the features... I tried to make the dpr header work in tests but it was too consuming :(